WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/FilterableCategoriesViewController.swift

```diff
@@ -31,6 +31,11 @@ class FilterableCategoriesViewController: CollapsableHeaderViewController {
     private var filteredSections: [CategorySection]?
     private var visibleSections: [CategorySection] { filteredSections ?? categorySections }
 
+    /// Should be overidden if a subclass uses different sized thumbnails.
+    var ghostThumbnailSize: CGSize {
```

Comment:
A use case for this is: I have a subclass and it has its own custom thumbnail size. When the content is loading I need to show ghost cells of the custom size, so I override this.